### PR TITLE
fix: separate out validators who are awaiting activation queue

### PR DESCRIPTION
**Pull request by wadealexc**
Related to core PR: https://github.com/Layr-Labs/eigenlayer-contracts/pull/668

### CODE REVIEW

**Comment by wadealexc** [comment]
image

simulated what this looks like for the `status` command

**Comment by wadealexc** [comment]
This also changes `credentials` - `credentials` is fetching different validators now.

**Comment by wadealexc** [comment]
See the diff here: https://github.com/Layr-Labs/eigenpod-proofs-generation/pull/124/files#diff-26423230dffc235a0f8411025a8fcbd1c8dbfcf4772808eaa6c9de8af5644b3cR469-R470

File: cli/core/utils.go [hunk 1]
```diff
@@ -197,7 +201,7 @@ func SortByStatus(validators map[string]Validator) ([]Validator, []Validator, []
 		return withdrawnValidators[i].Index < withdrawnValidators[j].Index
 	})
 
-	return inactiveValidators, activeValidators, withdrawnValidators
+	return awaitingActivationQueueValidators, inactiveValidators, activeValidators, withdrawnValidators
```
Comment (jbrower95) [inline cli/core/utils.go:204]: if you're gonna add a new return group to this function signature, i would probably put it at the end right?

File: cli/core/utils.go [hunk 2]
```diff
@@ -444,7 +466,8 @@ func SelectAwaitingCredentialValidators(
 		validatorInfo := validatorInfos[i]
 
 		if (validatorInfo.Status == ValidatorStatusInactive) &&
-			(validator.Validator.ExitEpoch == FAR_FUTURE_EPOCH) {
+			(validator.Validator.ExitEpoch == FAR_FUTURE_EPOCH) &&
+			(validator.Validator.ActivationEpoch != FAR_FUTURE_EPOCH) {
```
Comment (jbrower95) [inline cli/core/utils.go:470]: this is the actual change, yeah?

**Review by jbrower95 — APPROVED** [review]

File: cli/core/utils.go (hunk 1 above)
Comment (wadealexc) [reply cli/core/utils.go:204]: i was going for "lifecycle" here, since the left-most validators would occur earliest and then progress left to right

**Pull request merged**